conf.d/kubernetes.yaml.example

```diff
@@ -36,3 +36,10 @@ instances:
   #
   # enabled_gauges:
   #   - filesystem.*
+
+  # Kube State API
```

Comment:
We should also add `kube_state_url` and explain the deal with the kube state api in a detailed comment